circuits/circom/package.json

```
@@ -4,13 +4,13 @@
   "description": "Prove membership in a poseidon merkle tree by proving an ECDSA signature",
   "main": "",
   "scripts": {
-    "test": "bash maybe-fix-circom-imports.sh && jest"
+    "test": "export NODE_OPTIONS=\"--max-old-space-size=8192\"  && bash maybe-fix-circom-imports.sh && jest"
```

nice!
So that did the trick.

nit:
This will set `NODE_OPTIONS` as a global env var for the current session.
Is that what you want/need?
Could we just do
```suggestion
    "test": "bash maybe-fix-circom-imports.sh && NODE_OPTIONS=--max-old-space-size=8192 jest"
```
So that we increase the memory only for that particular run of that jest command only.

As a note, no need to quote `--max-old-space-size=8192` as it doesn't include shell special characters